force-app/main/default/classes/DMLOperations.cls

```diff
@@ -159,15 +163,19 @@ public with sharing class DMLOperations {
 	 * @return The Account that was upserted.
 	 */
 	public static Account upsertAccount(String accountName) {
-		// Query for the Account
 		List<Account> accounts = [SELECT Id FROM Account WHERE Name = :accountName];
-
-		// if the Account exists, update the description
-
-		// otherwise, create a new Account and set the description
-
-		// Upsert the Account
-		return null;
+         if(accounts.isEmpty() == false) {
```

Comment:
If you want to check that a boolean method return is false, add `!` in front of it 
`!accounts.isEmpty()`